packages/ckeditor5-image/tests/imagestyle/imagestyleui.js

```diff
@@ -440,6 +440,46 @@ describe( 'ImageStyleUI', () => {
 				}
 			} );
 		} );
+
+		it( 'should be enabled when at least one of the nested buttons are enabled', () => {
+			groups = groups.map( group => {
+				const items = group.view.toolbarView.items;
+
+				items.map( item => {
+					item.isEnabled = false;
+
+					return item;
+				} );
+
+				const activeButton = items.first;
+
+				activeButton.isEnabled = true;
+
+				return { ...group, activeButton };
+			} );
+
+			for ( const { buttonView } of groups ) {
+				expect( buttonView.isEnabled ).to.be.true;
+			}
```

Comment:
This could be way simpler (why `map()` and return `activeButton`?):

```suggestion
			for ( const group of groups ) {
				const groupItems = group.view.toolbarView.items;

				for ( const item of groupItems ) {
					item.isEnabled = false;
				}

				groupItems.first.isEnabled = true;
			}

			for ( const { buttonView, config } of groups ) {
				expect( buttonView.isEnabled, `Failing group name: "${ config.name }"` ).to.be.true;
			}
```

Reply:
I didn't want to mutate the `groups` and mess up the following tests. I missed the fact that each of these tests has its own `beforeEach` and thus separate `groups` array. Thanks for the catch :)